### PR TITLE
Added new configurations and updates Jobnames

### PR DESCRIPTION
**Pull request by Anipik**
Changing the job names caused the links on readme page to fail. This updates those links and adds new configuration that we added for fullframework like testing it on ubuntu and full framework

### CODE REVIEW

**Comment by Anipik** [comment]
To look at the Links just view the file in the changes tab. It shows to correct green tab for the links.

**Comment by shauheen** [comment]
Thanks @Anipik

File: README.md [hunk 1]
```diff
@@ -46,11 +46,13 @@ To build ML.NET from source please visit our [developers guide](docs/project-doc
...
+|**Windows FullFramework**|[![x64-debug](https://dnceng.visualstudio.com/public/_apis/build/status/dotnet/machinelearning/MachineLearning-CI?branchName=master&jobname=Windows_x64_NetFx461&configuration=Debug_Build)](https://dnceng.visualstudio.com/DotNet-Public/_build/latest?definitionId=104&branch=master)|[![x64-release](https://dnceng.visualstudio.com/public/_apis/build/status/dotnet/machinelearning/MachineLearning-CI?branchName=master&jobname=Windows_x64_NetFx461&configuration=Release_Build)](https://dnceng.visualstudio.com/DotNet-Public/_build/latest?definitionId=104&branch=master)|
+|**Windows x86**|[![Build Status](https://dnceng.visualstudio.com/public/_apis/build/status/dotnet/machinelearning/MachineLearning-CI?branchName=master&jobName=Windows_x86_Netcoreapp21&configuration=Debug_Build)](https://dnceng.visualstudio.com/public/_build/latest?definitionId=104?branchName=master)|[![Build Status](https://dnceng.visualstudio.com/public/_apis/build/status/dotnet/machinelearning/MachineLearning-CI?branchName=master&jobName=Windows_x86_Netcoreapp21&configuration=Release_Build)](https://dnceng.visualstudio.com/public/_build/latest?definitionId=104?branchName=master)|
```
Comment (codemzs) [inline README.md]: Doesn’t say debug or release here and below?

Comment (Anipik) [reply README.md]: Its mentioned in the build defination api url 
> branchName=master&jobName=Windows_x86_Netcoreapp21&configuration=Debug_Build)]

Comment (Anipik) [reply README.md]: I am not sure why the tags are different here for few of the links. Most probably, it seems like a comment. i will find out the reason

**Pull request merged**